Documentation/kubelet-wrapper.md

```diff
@@ -74,6 +74,20 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
   ...other flags...
 ```
 
+### Allow pods to use rbd volumes
+
+Pods using the [rbd volume plugin][rbd-example] to consume data from ceph must ensure that the kubelet has access to modprobe. Add the following options to the `RKT_OPTS` env before launching the kubelet via kubelet-wrapper:
+
+```ini
+[Service]
+Environment="RKT_OPTS=--volume modprobe,kind=host,source=/usr/sbin/modprobe \
+--mount volume=modprobe,target=/usr/sbin/modprobe \
+--volume lib-modules,kind=host,source=/lib/modules \
+--mount volume=lib-modules,target=/lib/modules \
+Environment=KUBELET_VERSION=v1.3.4_coreos.0
```

Comment:
Can we add a note about minimum image version that contains the ceph packages (which will be v1.3.6)